canon/CHANGELOG.md

```diff
@@ -0,0 +1,84 @@
+# 0.6.0
+
+## Changed
+
+Deprecate the usage of the `Store` trait.
+
+the trait `Canon<S>` becomes `Canon`.
+
+`Canon::<S>::write(&type, &mut Sink<S>)?;` becomes `type.encode(&mut sink);`
+
+Writes are assumed to always succeed, analgous to how a push to a `Vec` in rust can only panic with out of memory exceptions, but is assumed to be failsafe.
```

Comment:
This looks like more the body of a commit rather than the summary of it, we should have a "summary" here.